vagrant/dev/bootstrap.sh

```diff
@@ -12,6 +12,8 @@ yum install -y epel-release
 yum install -y https://s3.amazonaws.com/clusterhq-archive/centos/clusterhq-release$(rpm -E %dist).noarch.rpm
 
 # Install packages
+# XXX: We're disabling SELinux now so libselinux-devel may no longer be needed.
+# See https://clusterhq.atlassian.net/browse/FLOC-1643
```

Comment:
FLOC-1643 is being resolved now.  This comment should be resolved or this reference should point somewhere else.  Does the box build if you remove that package (does the box build at all - I think it's broken in other ways at the moment ...)?